bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -138,16 +167,18 @@ if args.make_signal_kde:
             logging.info('Using log param: %s', param)
             signal_pars.append(numpy.log(pvals))
         else:
-            raise ValueError("invalid log param argument, use 'True', or 'False'")
+            raise ValueError("invalid log param argument, use 'True', \
+                             or 'False'")
     
     signal_samples = numpy.vstack((signal_pars)).T
     logging.info('Starting optimization of signal KDE parameters')  
     optbw, optalpha = optimizedparam(signal_samples, nfold=args.nfold_signal)
     logging.info('Bandwidth %.4f, alpha %.2f' % (optbw, optalpha))
     logging.info('Evaluating signal KDE')
-    signal_kde = kde_awkde(signal_samples, temp_samples, alp=optalpha, gl_bandwidth=optbw)
-
-    f_dest.create_dataset("signal_kde", data=signal_kde)
-
+    signal_kde = kde_awkde(signal_samples, temp_samples, \
+                           alp=optalpha, gl_bandwidth=optbw)
+    f_dest.create_dataset("data_kde", data=signal_kde)
+    f_dest.attrs['stat'] = "signal-kde_file"
+  
 f_dest.close()
```

Comment:
so just before this I think you can add the optimal alpha and bw as attrs (this should work equally for template and signal kdes)